Gemfile

```diff
@@ -119,8 +119,6 @@ gem "select2-rails", github: "openfoodfoundation/select2-rails", branch: "v349_w
 
 gem 'ofn-qz', github: 'openfoodfoundation/ofn-qz', branch: 'ofn-rails-4'
 
-gem 'good_migrations'
```

Comment:
Hm, I thought that now that we updated to Zeitwerk 2.5 it would work again...

> Version 2.5 or higher of the Zeitwerk autoloader (e.g. config.autoloader = :zeitwerk) If your app uses an earlier version of zeitwerk, you'll see a warning every time db:migrate is run

https://github.com/testdouble/good-migrations/

Reply:
Ah I thought we were still stuck at `2.3`. I'll take another look. I had a lot of problems getting this to work, I think partly because `Spree::Preferences` already does a lot of _unusual_ class loading under the hood. Between Zeitwerk and good_migrations issues it was a real struggle :disappointed: